_data/health.yml

```diff
@@ -13,9 +14,10 @@ websites:
       facebook: carilionclinic
       email_address: direct@carilionclinic.org
 
-    - name: CVS/Caremark
-      url: https://www.caremark.com
-      twitter: cvshealth
+    - name: CVS Health
```

Comment:
The old version was for CVS Caremark which is a branch of CVS Health. The site you linked to doesn't have any login requirement whereas CVS Caremark does.